common/lib/xmodule/xmodule/js/karma_xmodule.conf.js

```diff
@@ -39,6 +39,7 @@ var options = {
         {pattern: 'common_static/js/src/utility.js', included: true},
         {pattern: 'common_static/js/test/add_ajax_prefix.js', included: true},
         {pattern: 'common_static/js/test/i18n.js', included: true},
+        {pattern: 'common_static/common/js/vendor/hls.js'},
```

Comment:
can we add this a little above near to vendor items ? something near to 'common_static/js/vendor/tinymce/js/tinymce/tinymce.full.min.js'